textattack/shared/attacked_text.py

```diff
@@ -231,24 +282,28 @@ def replace_new_words(self, new_words):
             with multiple space-separated words, representation an insertion
             of one or more words.
         """
-        final_sentence = ""
-        text = self.text
+        perturbed_text = ""
+        original_text = AttackedText.SPLIT_TOKEN.join(self._text_input.values())
```

Comment:
did you test inserting at the end of the premise and beginning of hypothesis? feels like the split token could mess this up

Reply:
Also maybe current_text is a better tname than original_text?

Reply:
It feels weird that we still have to use the SPLIT_TOKEN here